recipes-bsp/u-boot/u-boot-starfive_v2021.04.bb

```diff
@@ -3,7 +3,7 @@ require recipes-bsp/u-boot/u-boot.inc
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-SRC_URI = "git://github.com/starfive-tech/u-boot.git;protocol=git;branch=Fedora_VIC_7100_2021.04 \
+SRC_URI = "git://github.com/atishp04/u-boot.git;protocol=git;branch=beagleV/Fedora_VIC_7100_2021.04 \
```

Comment:
why should we switch to this fork ?

Reply:
Apparently that's the one to use: https://github.com/starfive-tech/opensbi/issues/4#issuecomment-852488314

Reply:
OK and I hope this is short lived and eventually all these changes will flow into upstream u-boot ?

Reply:
That's my hope. It seems to only be two patches on top of the starfive branch, so hopefully we can switch to upstream or starfive soon.